_resources/port1.0/group/python-1.0.tcl

```diff
@@ -236,10 +245,35 @@ default destroot.destdir {"--prefix=[python_get_defaults setup_prefix] --root=${
 proc python_get_defaults {var} {
     global python.version python.branch prefix python.prefix
     switch -- $var {
+        branch {
+            if {[string match pypy* ${python.version}]} {
+                return ${python.version}
+            } else {
+                return [string range ${python.version} 0 end-1].[string index ${python.version} end]
+            }
+        }
+        group {
+            if {[string match pypy* ${python.version}]} {
+                return ${python.version}
+            } else {
+                return py[string map {. ""} ${python.version}]
```

Comment:
Group seems like a strange name for this. Maybe it would make more sense if its meaning and purpose were explained though.